auto_tuning/proxy/src/poxy_main.cpp

```diff
@@ -0,0 +1,40 @@
+#include <utils/args.h>
+#include "proxy_common.hpp"
+#include<iostream>
+
+ProxyOutput runProxy(ProxyConfig config);
```

Comment:
Why do you not add it to proxy_common.hpp or another header?

Reply:
done